lib.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-use-before-define */
+const myLibrary = [];
+const books = document.getElementById('book_list');
+
+const form = document.getElementById('forms');
+const read1 = document.getElementById('read1');
+
+const newBook = document.getElementById('addBook');
+
+function Book(name, author, pages, read) {
+  this.name = name;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+function deletethisBook(i, myLibrary) {
+  myLibrary.splice(i, 1);
+  displayLibrary(myLibrary);
+}
+
+function changeStatus(i, myLibrary) {
+  myLibrary[i].read = !myLibrary[i].read;
```

Comment:
- [ ] There seems to be a bug on this line. When I try to create a new book, it returns an error in the console `Uncaught TypeError: Cannot read property 'read' of undefined`. It will be great to fix this bug so that your application can function as expected. ⬇️ 
![read undefined](https://user-images.githubusercontent.com/32361963/128153346-8f282bb5-ef4b-4550-aa5d-09afaf412cc3.png)